code/headers/module_task.hpp

```diff
@@ -23,6 +23,12 @@ namespace r2d2::module_scheduler {
         arena_alloc_c<WaitableAllocSize> allocator;
 
     public:
+        // initializes the module and it's task
```

Comment:
it's should be its

Reply:
https://github.com/R2D2-2019/module_scheduler/pull/5/commits/fbc15c836927b0f79866df015a2da0bd959d0550